src/SFA.DAS.EmployerFinance.Jobs/Program.cs

```diff
@@ -7,64 +7,72 @@
 using Microsoft.Extensions.Logging;
 using NLog.Extensions.Logging;
 using SFA.DAS.AutoConfiguration;
+using SFA.DAS.EmployerFinance.Configuration;
 using SFA.DAS.EmployerFinance.Jobs.DependencyResolution;
 using SFA.DAS.EmployerFinance.Startup;
+using StructureMap;
 
 namespace SFA.DAS.EmployerFinance.Jobs
 {
     public static class Program
     {
-        private const string AppSettingFilePath = "appsettings.json";
-
         public static async Task Main()
         {
             ServicePointManager.DefaultConnectionLimit = 50;
 
             using (var container = IoC.Initialize())
             {
-                var startup = container.GetInstance<IStartup>();
+                var environmentService = container.GetInstance<IEnvironmentService>();
 
-                await startup.StartAsync();
+                var environmentSettings = HostEnvironmentSettingsFactory.Create(environmentService);
```

Comment:
Can the factory be resolved from the container instead?

Reply:
sure

Reply:
I've actually removed this now as its no longer needed due to the changes mentioned about the start up code